### DISCUSSION TITLE
Integrate with metabase

### DISCUSSION THREAD

**Discussion by chsasank** [RamiAwar/dataline#359] [Ideas]
Metabase is one of the most popular tool for BI. It has fairly generic database interface and has easy UI to edit a query. AI is never gonna do the full job anyway, so it's better if we can provide UI that lets user edit after the first iteration. Metabase is a good tool for this sort of thing.

If we can agree on a sort of architecture of this feature, I am happy to work on this and make a PR :)

**Comment by RamiAwar** [comment]
I'm all for supporting this! @chsasank 

I don't know much about Metabase unfortunately, but I have heard of it from a friend who also suggested this 🙂 

How are you imagining doing this? I'd be happy to dedicate some time to think this out with you 🙂
